### PR TITLE
Add opentracing utils

### PR DESCRIPTION
**Pull request by elauria**
#131

### CODE REVIEW

File: client/bower.json [hunk 1]
```diff
@@ -32,7 +32,9 @@
     "paper-avatar": "Abe90/paper-avatar#^1.0.4",
     "web-animations-js": "^2.3.1",
     "paper-dropdown": "https://github.com/elauria/paper-dropdown.git#^2.0.1",
-    "iron-autogrow-textarea": "^2.1.0"
+    "iron-autogrow-textarea": "^2.1.0",
+    "opentracing-javascript-utils": "https://github.com/zalando-zmon/opentracing-javascript-utils.git#^1.0.0",
+    "lightstep-overlay": "https://github.com/lightstep/lightstep-overlay.git"
```
Comment (mohabusama) [inline client/bower.json]: Can we remove vendor specific libs from OSS and may be find a way to include in the browser post-deployment? May be via a config and using a CDN link?

**Comment by mohabusama** [comment]
@elauria this is mainly Stackenblochen in me kicking in, but I don't like vendor being mentioned in commits and configs :(

File: client/bower.json (hunk 1 above)
Comment (elauria) [reply client/bower.json]: yes! just keeping it here for now to see it working but thanks for reminding me!

**Comment by elauria** [comment]
👍

**Comment by elauria** [comment]
👍

**Comment by mohabusama** [comment]
👍

**Pull request merged**